website/payments/signals.py

```diff
@@ -21,3 +21,32 @@ def give_new_users_tpay_permissions(instance, created, **kwargs):
         # after this signal call (triggered by the model's `save()`), a
         # `save_related()` will be called which will override the permissions
         # again.
+
+
+@suspendingreceiver(post_save, sender="payments.BankAccount")
+def clear_enabled_cache(sender, instance, **kwargs):
```

Comment:
Is there another way to trigger this, for example if we change the global settings?

Reply:
Maybe a management command that is executed on startup?